portfolio/src/main/webapp/index.html

```diff
@@ -7,9 +7,20 @@
     <script src="script.js"></script>
   </head>
   <body>
+
+    <form action="/form-handler" method="POST">
+        <p>Enter some text here:</p>
+        <textarea name="text-input">hello world</textarea>
+        <br/>
+        <input type="submit" />
+      </form>
```

Comment:
Hm. I don't see this on your site https://echen-sps-summer22.appspot.com/, is it up to date?